Initial base for AltFP (#2)
riscv/riscv-alt-fp: doc/riscv-bfloat16-spec.adoc

have helped to improve this specification through their comments, reviews,
feedback and questions.

// ------------------------------------------------------------

include::riscv-bfloat16-introduction.adoc[]
include::riscv-bfloat16-audience.adoc[]
include::riscv-bfloat16-format.adoc[]
// include::riscv-bfloat16-sail-specifications.adoc[]
// include::riscv-bfloat16-policies.adoc[]

// ------------------------------------------------------------

[[bfloat16_extensions]]
== Extensions Overview

The group of extensions introduced by the BFloat16 Instruction Set
Extension is listed here.

Detection of individual BFloat16 extensions uses the
unified software-based RISC-V discovery method.

[NOTE]
====
At the time of writing, these discovery mechanisms are still a work in
progress.
====

.A note on extension rationale
[NOTE, caption="SH"]
====
BFloat16 instructions are separated into different
functional groups because not all implementations will require all
of the functionality.

====

// include::riscv-bfloat16-zfbmin.adoc[]

=== `Zfbmin` - BFloat16 minimal 

This extension provides the minimal support required for the BFloat16

> format. It enables BFloat16 to be an interchange format whereby it

Reviewer: I would go either way
```suggestion
format. It enables BFloat16 as an interchange format whereby it
```
